src/single.test.js

```diff
@@ -21,28 +21,28 @@ describe("BStack demo test", () => {
     await driver.manage().window().maximize();
   }, 100000);
 
-  test("login test", async () => {
-    await driver.get("https://bstackdemo.com");
-
-    await driver.findElement(By.css("#signin")).click();
-
-    await driver.wait(until.elementLocated(By.css("#username input")));
-
-    await driver
-      .findElement(By.css("#username input"))
-      .sendKeys("locked_user", Key.ENTER);
+  test("add items to cart", async () => {
+    await driver.get("https://bstackdemo.com/");
+    await driver.wait(until.titleMatches(/StackDemo/i), 10000);
 
-    await driver
-      .findElement(By.css("#password input"))
-      .sendKeys("testingisfun99", Key.ENTER);
-
-    await driver.findElement(By.css("#login-btn")).click();
-
-    await driver.wait(until.elementLocated(By.css(".api-error")));
-
-    expect(await driver.findElement(By.css(".api-error")).getText()).toBe(
-      "Your account has been locked."
-    );
+    // locating product on webpage and getting name of the product
+    let productText = await driver
+      .findElement(By.xpath('//*[@id="1"]/p'))
+      .getText();
+    // clicking the 'Add to cart' button
+    await driver.findElement(By.xpath('//*[@id="1"]/div[4]')).click();
+    // waiting until the Cart pane has been displayed on the webpage
+    driver.findElement(By.className("float-cart__content"));
+    // locating product in cart and getting name of the product in cart
+    let productCartText = await driver
+      .findElement(
+        By.xpath(
+          '//*[@id="__next"]/div/div/div[2]/div[2]/div[2]/div/div[3]/p[1]'
```

Comment:
no other way than xpath? some className or id?

Reply:
I was asked to make sure the tests are exactly as they are in other documentation repositories and refer to [this](https://github.com/browserstack/junit-browserstack/blob/master/junit-4/src/test/java/com/browserstack/SingleTest.java) for the elements and locators.